packages/components/README.md

```diff
@@ -1,17 +1,27 @@
 # @contentful/experiences-components-react
+### Purpose
+- Stores React specific definitions for Structure and Built-in copmonents provided to the user by Contentful.
+- Note that this is a React specific package. Therefore, unlike the core package, React specific compatibilities are implemented here and support for other frontend frameworks would result in a new package being created under a separate namespace.
 
-This package provides the default structure and basic components for the [Experiences SDK](https://www.contentful.com/developers/docs/experiences/set-up-experiences-sdk/). You can use these components directly to start building your experiences, or as examples to create your own components.
+### Concepts
+- **Structure component**: Contentful provided components for users to define their layout in the visual editor canvas.
```

Comment:
Contentful-provided + maybe instead of the "visual editor canvas" use "experience canvas" or "Experiences canvas"?